accern_xyme/accern_xyme.py

```diff
@@ -751,6 +753,44 @@ def get_settings(self) -> Dict[str, Any]:
         assert self._settings is not None
         return self._settings
 
+    def get_timing(
+                self,
+                blacklist: List[str] = [],
+                ) -> Optional[TimingResult]:
+        node_timing: Dict[str, NodeTiming] = {}
+        nodes = self.get_nodes()
+
+        def filter_blacklist(node_time: List[Timing]) -> Iterator[Timing]:
+            for value in node_time:
+                if value["name"] not in blacklist:
+                    yield value
+
+        for node in nodes:
+            node_time = self.get_node(node).get_timing()
+            node_name: str = self.get_node(node).get_node_def()["name"]
+            node_total = 0.0
+            for cur in filter_blacklist(node_time):
+                length: int = len(node_time)
+                node_id = self.get_node(node).get_id()
+                node_sums = float(cur["total"])
+                node_total += (node_sums)
+                node_timing[node_id] = {
```

Comment:
so you are assigning node_timing inside the loop -- it gets overwritten each time -- move it out of the inner loop